shared/ui-components/src/headers/SiteHeader/SiteHeader.tsx

```diff
@@ -53,7 +53,7 @@ class SiteHeader extends React.Component<SiteHeaderProps, SiteHeaderState> {
     )
   }
 
-  handleMenuToggle() {
+  handleMenuToggle = () => {
```

Comment:
@jaredcwhite This is a weird-to-me pattern that the linter wants so that there's not a reference problem when you call bind() that I don't completely understand. I don't think I broke anything. See also https://palantir.github.io/tslint/rules/no-unbound-method/, and maybe you can explain it to me.

Reply:
@bk3c Looks weird, but it seems to work so let's roll with it. Merge away! (I also removed a couple extra file references from index.scss…holdover from the previous merge.)